packages/api/tests/endpoints/test-executions.js

```diff
@@ -795,3 +829,154 @@ test.serial('POST /executions/workflows-by-granules returns correct workflows wh
 
   t.deepEqual(response.body.sort(), ['fakeWorkflow', 'workflow2']);
 });
+
+test.serial('POST an api execute record creates a new execution in Dynamo and PG with correct timestamps', async (t) => {
+  const newExecution = fakeExecutionFactoryV2({
+    asyncOperationId: t.context.testAsyncOperation.id,
+    collectionId: t.context.collectionId,
+    parentArn: t.context.fakeApiExecutions[1].arn,
+  });
+
+  const expectedPgRecord = await translateApiExecutionToPostgresExecution(
+    newExecution,
+    t.context.knex
+  );
+
+  await request(app)
+    .post('/executions')
+    .send(newExecution)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  const fetchedDynamoRecord = await executionModel.get({
+    arn: newExecution.arn,
+  });
+
+  const fetchedPgRecord = await executionPgModel.get(
+    t.context.knex,
+    {
+      arn: newExecution.arn,
+    }
+  );
+
+  t.true(fetchedDynamoRecord.createdAt > newExecution.createdAt);
+  t.true(fetchedDynamoRecord.updatedAt > newExecution.updatedAt);
+
+  // PG and Dynamo records have the same timestamps
+  t.is(fetchedPgRecord.created_at.getTime(), fetchedDynamoRecord.createdAt);
+  t.is(fetchedPgRecord.updated_at.getTime(), fetchedDynamoRecord.updatedAt);
+
+  t.is(fetchedPgRecord.arn, fetchedDynamoRecord.arn);
+  t.truthy(fetchedPgRecord.cumulus_id);
+  t.truthy(fetchedPgRecord.async_operation_cumulus_id);
+  t.truthy(fetchedPgRecord.collection_cumulus_id);
+  t.truthy(fetchedPgRecord.parent_cumulus_id);
+
+  const omitList = ['createdAt', 'updatedAt', 'created_at', 'updated_at', 'cumulus_id'];
+  t.deepEqual(
+    omit(fetchedDynamoRecord, omitList),
+    omit(newExecution, omitList)
+  );
+  t.deepEqual(
+    omit(fetchedPgRecord, omitList),
+    omit(expectedPgRecord, omitList)
+  );
```

Comment:
```suggestion
  t.like(
    fetchedDynamoRecord,
    newExecution
  );
  t.like(
    fetchedPgRecord,
    expectedPgRecord
  );
```

`t.like` will only compare properties that exist on both objects

Reply:
`newExecution` and `expectedPgRecord` have the properties on the omitList like 'createdAt', 'updatedAt', 'created_at', 'updated_at'